.travis.yml

```diff
@@ -55,6 +55,44 @@ env:
   - CHUNK="3/4"
   - CHUNK="4/4"
 
+# General environment setup before we start installing stuff
```

Comment:
I think the `.travis.yml` was changed in the past in order to install some library. But it was done in one of the scripts called from `.travis.yml`. I believe there are no changes required in the Travis set up for this pull request, so this change can be discarded?

Reply:
I added a dev dependency in .travis.yml
`# Testfixtures
    - pip install testfixtures` I should discard this?

Reply:
Yup, because that didn't work. I think in Riot we found out that it was not being run. The reason is that while you were working on this issue, I changed the `.travis.yml` file, and used external scripts for some tasks. The `install` task is in a shell file. Take a look at the `.travis.yml` from [master here](https://github.com/cylc/cylc/blob/697c3c2ecbd7b34c2682caf74aae9206e6772450/.travis.yml#L59). 

The file that needed to be updated is ` .travis/install.sh`, and you did that (scroll down if you are reading this in GitHub). What we are seeing here, I believe, is the left-overs of your initial change, that cause this diff because the master branch was changed too (sorry about that :disappointed: ). I think `.travis.yml` doesn't need to be changed, only ` .travis/install.sh` (good to double check too if you added the dependency for unit tests, for functional tests, or for both.. not sure which one needs it, but I guess only unit?).